### ISSUE TITLE
Weird build numbers on the the Myget feed causing failures?

### ISSUE THREAD

**Issue by davidfowl** [dotnet/corefxlab#450]
https://www.myget.org/F/dotnet-corefxlab/api/v3/flatcontainer/System.Buffers/index.json

I tried to use it in my `project.json`

```JSON
{
    "version": "1.0.1-*",
    "compilationOptions": {
        "emitEntryPoint": true
    },
    "dependencies" : {
        "System.Buffers": "0.1.0-*",
        "Microsoft.AspNet.Server.Kestrel": "1.0.0-*",
        "Microsoft.NETCore.Platforms": "1.0.1-beta-23504"
    },
    "frameworks": {
        "dnxcore50": { }
    }
}
```

Ended up with strange errors:

```
Warning: DownloadPackageAsync: https://www.myget.org/F/dotnet-corefxlab/api/v3/flatcontainer/system.buffers/0.1.0-d7iledaae-1/system.buffers.0.1.0-d7iledaae-1.nupkg
  Response status code does not indicate success: 404 (No package matching package identifier system.buffers and version 0.1.0-d7iledaae-1 was found in the given feed.).
  GET https://www.myget.org/F/dotnet-corefxlab/api/v3/flatcontainer/system.buffers/0.1.0-d7iledaae-1/system.buffers.0.1.0-d7iledaae-1.nupkg
Warning: DownloadPackageAsync: https://www.myget.org/F/dotnet-corefxlab/api/v3/flatcontainer/system.buffers/0.1.0-d7iledaae-1/system.buffers.0.1.0-d7iledaae-1.nupkg
  Response status code does not indicate success: 404 (No package matching package identifier system.buffers and version 0.1.0-d7iledaae-1 was found in the given feed.).
  GET https://www.myget.org/F/dotnet-corefxlab/api/v3/flatcontainer/system.buffers/0.1.0-d7iledaae-1/system.buffers.0.1.0-d7iledaae-1.nupkg
Error: DownloadPackageAsync: https://www.myget.org/F/dotnet-corefxlab/api/v3/flatcontainer/system.buffers/0.1.0-d7iledaae-1/system.buffers.0.1.0-d7iledaae-1.nupkg
  Response status code does not indicate success: 404 (No package matching package identifier system.buffers and version 0.1.0-d7iledaae-1 was found in the given feed.).
```

Might be a myget bug /cc @maartenba or a nuget/dnu bug

**EDIT** This is the only version that works **0.1.0-d103015-1**

**Comment by maartenba** [comment]
It's because of the uppercase characters in version number. Will add a fix.

**Comment by maartenba** [comment]
Ideally packages have lowercase build numbers altogether, but this is definitely a bug on our side. Fix is ready for deployment, may deploy later today or halfway this week.

**Comment by maartenba** [comment]
Fix deployed. Can you confirm it's working as expected now?

**Comment by sokket** [comment]
I also put in a fix to prevent those wonky build numbers. I nuked the myget feed since there were actually some invalid packages due to some bug I haven't tracked down yet...I'll be pushing new packages with correct version numbers shortly

**Comment by sokket** [comment]
This was fixed by #460
